Mathlib/Computability/ContextFree.lean

```diff
@@ -6,14 +6,18 @@ Authors: Martin Dvorak
 import Mathlib.Computability.Language
 
 /-!
-# Context-Free Grammar
+# Context-Free
```

Comment:
what happened to the word grammar, this seems like an odd title now

Reply:
I renamed the file. Is it mandatory to change the "docs headline" with it?

Reply:
I should have checked other files. My bad!